### PR TITLE
Bug | Duplicate records created when adding phases via keyboard #6016

### PR DESCRIPTION
**Pull request by sergiogcx**
Closes https://github.com/cityofaustin/atd-data-tech/issues/6016

** Front-End Changes Only, Live Demo:
https://deploy-preview-307--atd-moped-main.netlify.app/moped/session/signin

### CODE REVIEW

File: moped-editor/src/views/projects/projectView/ProjectTimeline.js [hunk 1]
```diff
@@ -114,7 +152,7 @@ const ProjectTimeline = () => {
   const phaseNameLookup = data.moped_phases.reduce(
     (obj, item) =>
       Object.assign(obj, {
-        [item.phase_name]:
+        [item.phase_name.toLowerCase()]:
```
Comment (sergiogcx) [inline moped-editor/src/views/projects/projectView/ProjectTimeline.js:155]: It's here where we fix the empty phases name bug

File: moped-editor/src/views/projects/projectView/ProjectTimeline.js [hunk 2]
```diff
@@ -106,6 +65,85 @@ const ProjectTimeline = () => {
...
 
+  /**
+   * Prevents the line from being saved on enter key
+   * @param {object} e - Event Object
+   */
+  const handleKeyEvent = e => {
+    if (e.key === "Enter" || e.key === " ") {
+      e.stopPropagation();
+    }
+  };
```
Comment (sergiogcx) [inline moped-editor/src/views/projects/projectView/ProjectTimeline.js:76]: This is how we capture the enter or space events on the keyboard

**Review by chiaberry — APPROVED** [review]
👍🏼  yay

**Review by mateoclarke — APPROVED** [review]
🚢

**Review by amenity — APPROVED** [review]
Nicely done, @sergiogcx !! 

(And good catch, @tillyw 🕵️‍♀️ 🐛)

**Pull request merged**